CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController+ToolTip.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Camera2ViewController+Extension.swift
```

Comment:
[Suggestion] File is named `...+ToolTip`, but it is used to show the QR code detected popup. As we won't have tooltips anymore I recommend to use `...+Popups` instead. Oh and also change the file name in the comment here 😉.